scheduler/src/cook/prometheus_metrics.clj

```diff
@@ -0,0 +1,117 @@
+                                 {:description "Distribution of generating matches->job-uuids map latency"
+                                  :labels [:pool]
+                                  :quantiles default-summary-quantiles})
+             (prometheus/summary :cook/scheduler-launch-all-matched-tasks-total-duration-seconds
+                                 {:description "Distribution of total launch all matched tasks latency"
+                                  :labels [:pool]
+                                  :quantiles default-summary-quantiles})
+             (prometheus/summary :cook/scheduler-launch-all-matched-tasks-transact-duration-seconds
+                                 {:description "Distribution of launch all matched tasks--transact in datomic latency"
+                                  :labels [:pool]
+                                  :quantiles default-summary-quantiles})
+             (prometheus/summary :cook/scheduler-launch-all-matched-tasks-submit-duration-seconds
+                                 {:description "Distribution of launch all matched tasks--submit to compute cluster latency"
+                                  :labels [:pool]
+                                  :quantiles default-summary-quantiles})
+             (prometheus/summary :cook/scheduler-trigger-autoscaling-duration-seconds
+                                 {:description "Distribution of trigger autoscaling latency"
+                                  :labels [:pool]
+                                  :quantiles default-summary-quantiles})
+             (prometheus/summary :cook/scheduler-kill-cancelled-tasks-duration-seconds
+                                 {:description "Distribution of kill cancelled tasks latency"
+                                  :labels [:pool]
+                                  :quantiles default-summary-quantiles})
+             (prometheus/summary :cook/scheduler-sort-jobs-hierarchy-duration-seconds
+                                 {:description "Distribution of sorting jobs by DRU latency"
+                                  :labels [:pool]
+                                  :quantiles default-summary-quantiles})
+             (prometheus/summary :cook/scheduler-filter-offensive-jobs-duration-seconds
+                                 {:description "Distribution of filter offensive jobs latency"
+                                  :quantiles default-summary-quantiles})
+             (prometheus/summary :cook/scheduler-handle-status-update-duaration-seconds
+                                 {:description "Distribution of handle compute cluster status update latency"
+                                  :quantiles default-summary-quantiles})
+             (prometheus/summary :cook/scheduler-handle-framework-message-duration-seconds
+                                 {:description "Distribution of handle framework message latency"
+                                  :quantiles default-summary-quantiles}))))
+
+(defn get-collector
```

Comment:
Re: making it easy to refer to collectors, I settled on using declared vars as the collector names.

We discussed combining the spans offline, and decided it not to do it since the co-existence of the prometheus and codahale metrics will be a temporary state.